src/argparse.cpp

```diff
@@ -2,15 +2,21 @@
 #include <iostream>
 #include <fstream>
 #include "cap32.h"
+#include "argparse.h"
 #include "stringutils.h"
 
 const struct option long_options[] =
 {
-   {"version", no_argument, nullptr, 'V'},
-   {"help",    no_argument, nullptr, 'h'},
-   {nullptr, 0,  nullptr, 0},
+   {"cfg_file", required_argument, nullptr, 'c'},
+   {"version",  no_argument, nullptr, 'V'},
+   {"help",     no_argument, nullptr, 'h'},
+   {nullptr, 0, nullptr, 0},
 };
 
+void initArgs(struct capriceArgs& args)
```

Comment:
The initArgs function is not needed: the default constructor of the string will set it to an empty string.
If this wasn't the case (or if we wanted to use different default values), making capriceArgs a class with a constructor would be better (no risk of forgetting to call init).